src/js/i18n/en-US.yml

```diff
@@ -239,9 +239,9 @@ careOptsFrontend:
           title: Schedule for { owner }
           tooltip: Schedule Worklist displays all Actions with a due date for you (or someone else on your team). States included are To Do and In Progress.
         tableHeaderView:
-          actionHeader: Action
           dueDateHeader: Due Date
-          statePatientHeader: State, Patient
+          patientHeader: Patient
+          stateActionHeader: State, Action
           formheader: Form
```

Comment:
Did a few things here besides changing the header label text:

* Renamed `statePatientHeader` to `patientHeader`.
* Renamed `actionHeader` to `stateActionHeader`.
* Re-ordered the variables to match the order they're used in the view file.

But now that I look at it again, does `stateActionHeader` make sense? It could just be `actionHeader`...?

Reply:
ah well it should be alphabetical.   I wonder if we could fix that in the loader somehow..  It doesn't _really_ matter, but if we ever use phraseapp again for translation, it'll alphabetize the keys.  `actionHeader` is fine with me.

Reply:
Ah gotcha. I switched it to `actionHeader` and alphabetized them.